build/Dockerfile.tz

```diff
@@ -0,0 +1,4 @@
+FROM registry.access.redhat.com/ubi8/ubi-minimal:latest
```

Comment:
Why don't you use the current Dockerfile?

Reply:
Initially I thought I will build an image call ubi-minimal-with-tz and then use that image in the 2 docker files. But sure. I can modify the existing docker files itself. Not a problem.